src/overlays/actors/ovl_Door_Shutter/z_door_shutter.c

```diff
@@ -773,18 +846,21 @@ void func_809973E8(DoorShutter* this, PlayState* play) {
         Quake_SetPerturbations(quakeIndex, 2, 0, 0, 0);
         Quake_SetDuration(quakeIndex, 10);
         Rumble_Request(this->dyna.actor.xyzDistToPlayerSq, 180, 20, 100);
-        func_80997220(this, play);
+        DoorShutter_SetupClosed(this, play);
     }
 }
 
+/**
+ * The Jabu door is being closed.
+ */
 void DoorShutter_JabuDoorClose(DoorShutter* this, PlayState* play) {
     if (Math_StepToS(&this->jabuDoorClosedAmount, 100, 10)) {
-        func_80997220(this, play);
+        DoorShutter_SetupClosed(this, play);
     }
 }
 
-void func_80997568(DoorShutter* this, PlayState* play) {
-    if (this->unk_16F++ > 30) {
+void DoorShutter_WaitPlayerShock(DoorShutter* this, PlayState* play) {
```

Comment:
`PlayerShock` made me think this was related to an electric shock in some way. Presumably this is when the player turns around, and surprised about the door being locked? Maybe
`DoorShutter_WaitPlayerSurpriseTurnAroundAnim`? Bit verbose though, might not need all those words

Reply:
Probably WaitPlayerSurprised is enough and indeed less ambiguous 👍 ?